numpy/core/src/multiarray/usertypes.c

```diff
@@ -176,6 +202,11 @@ PyArray_RegisterDataType(PyArray_Descr *descr)
         PyErr_SetString(PyExc_ValueError, "missing typeobject");
         return -1;
     }
+
+    if (test_deprecated_arrfuncs_members(f) < 0) {
```

Comment:
Put into its own helper in anticiapation of more functions joining. In particular `fastclip` is already deprecated at execution time, and should be added here as well now (as well as get the same documentation updates). Planning to do that once this is in.